gensim/models/fasttext.py

```diff
@@ -161,7 +161,7 @@ def train_batch_sg(model, sentences, alpha, work=None, neu1=None):
 
 class FastText(BaseWordEmbeddingsModel):
     """Class for training, using and evaluating word representations learned using method
-    described in [1]_ aka Fasttext.
+    described in [1] aka Fasttext.
```

Comment:
about `[1]_` bug, let's switch to `some text <http://asdfasd.com>`_ format (this is better than the link that doesn't work), here and everywhere.

Reply:
Ok!